### PR TITLE
improve top level docs

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

**Comment by nslater** [comment]
I changed the recommendation from Java to 7 to Java 8 to match the main CrateDB repos. also made a few other changes for readability that you might wanna check for technical accuracy

File: DEVELOP.rst [hunk 1]
```diff
@@ -1,243 +1,93 @@
...
-    ./gradlew test
+    $ ./gradlew test
 
 Integrations Tests
 ------------------
 
-Integration test suite can be run using the following Gradle command::
+The integration tests can be run like so::
 
     ./gradlew itest
```
Comment (chaudum) [inline DEVELOP.rst]: add `$` line prefix for consistency

File: README.rst [hunk 2]
```diff
@@ -11,123 +11,153 @@ Crate Mesos Framework
...
 
-The documentation for this project has moved into the ``docs/`` folder of this
-repository. It is built on ReadTheDocs where it can be found `here`_.
+Clone the repository::
+
+    $ git clone https://github.com/crate/crate-mesos-framework.git
+
+Build the JAR file::
+
+    ./gradlew fatJar
```
Comment (chaudum) [inline README.rst]: `$` line prefix

File: README.rst [hunk 3]
```diff
@@ -11,123 +11,153 @@ Crate Mesos Framework
...
+
+Launching
+=========
+
+This project ships with a Vagrantfile that can be used with Vagrant to launch
+*virtual machines* (VM) with Mesos installed.
+
+Launch the VMs like so::
+
+    vagrant up
```
Comment (chaudum) [inline README.rst]: `$` line prefix

Comment (chaudum) [inline README.rst]: please check all ocurrences again

**Pull request merged**